scripts/ci/pre_commit/checkout_no_credentials.py

```diff
@@ -57,6 +57,13 @@ def check_file(the_file: Path) -> int:
                     # build. This is ok for security, because we are pushing it only in the `main` branch
                     # of the repository and only for unprotected constraints branch
                     continue
+                if step.get("id") == "checkout-for-backport":
```

Comment:
@potiuk have updated this to ignore check, backport script require this option. as it creates backport branch and pr. tried with `persist-credentials: false` but not working..

Please suggest if there is any alternative or kindly check the permissions if i am missing anything :)

Reply:
This is good.